torchvision/transforms/transforms.py

```diff
@@ -32,6 +32,36 @@
 }
 
 
+class FillTransform(torch.nn.Module):
```

Comment:
Let's avoid introducing this new class. 

> Unlike Python, each variable in TorchScript function must have a single static type 
> typing.Union : Unlikely to be implemented

https://pytorch.org/docs/stable/jit_language_reference.html#language-reference